Add compile time errors on interop calls of constructors of abstract classes as breaking changes (#9050)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

- To avoid clashes with Java identifiers, the character used for encoding and decoding identifiers has been changed from `$` to `&`.

> - Interop calls to constructors of Java abstract classes has been restricted at compile time.

Reviewer: ``` suggestion
- Interop calls to Java abstract class constructors have been restricted at compile time.
```

> - A bug which resulted in no compilation errors for interop calls of Java abstract class constructors has been fixed.

Reviewer: ```suggestion
- A bug that resulted in no compilation errors for interop calls of Java abstract class constructors has been fixed.
```

Reviewer 2: Should we mention interfaces as well

Author: Yes, it's applicable for interfaces too.

> - A bug which resulted in no compilation errors for interop calls of Java abstract class constructors has been fixed.

Reviewer 3: ```suggestion
- A bug that allowed interop calls of Java abstract class constructors to proceed without generating compilation errors has been fixed.
```
This sounds a bit more explanatory. Is it too long?

Author: Stick with the original sentence to follow the writing style of this doc.
